  Header header
  uint8 motor 
  uint8 status_bits
  uint8 job_id
  uint32 pulses_remaining
